FWCore/Skeletons/python/utils.py

```diff
@@ -101,6 +101,7 @@ def capture():
         print("\n### generated code\n")
         print(func)
     # compile python code as exec statement
+    print(func)
```

Comment:
Why do we need this line also outside debug? If need, should we move it out from debug (line 102, above)?

Reply:
good question. I'm not sure how I added that, but it was not intended. Fixing that.